WeakAuras/GenericTrigger.lua

```diff
@@ -4428,6 +4428,15 @@ WeakAuras.GetHitChance = function()
   return max(melee, ranged, spell)
 end
 
+Private.GetCurrencyInfoForTrigger = function(trigger)
```

Comment:
I've changed how the options work, in that "relevant" checks are displayed depending on the currency selected.

There are e.g. 3 different ways a currency can be capped: "quantity/maxQuantity", "quantityEarnedThisWeek/maxWeeklyQuantity", and "totalEarned/maxQuantity". The options and conditions now offer a check if it's applicable to the selected currency.

That check is a "tristate", so the user can check if "At cap", "Not at cap" or ignore that check.